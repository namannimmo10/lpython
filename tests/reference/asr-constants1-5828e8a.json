{
    "basename": "asr-constants1-5828e8a",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/constants1.py",
    "infile_hash": "ca0a78d7946779e18980611ff4b95bf6123e2fba6f641463cbe70583",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-constants1-5828e8a.stdout",
    "stdout_hash": "4224d133210fdf6e0a704125c9514c6dca59914c895adb12769d38b3",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}