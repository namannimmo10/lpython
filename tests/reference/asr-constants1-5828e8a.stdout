(TranslationUnit (SymbolTable 1 {test_abs: (Subroutine (SymbolTable 4 {a: (Variable 4 a Local () () Default (Integer 4 []) Source Public Required .false.), b: (Variable 4 b Local () () Default (Real 4 []) Source Public Required .false.)}) test_abs [] [(= (Var 4 a) (ConstantInteger 5 (Integer 4 [])) ()) (= (Var 4 a) (ConstantInteger 0 (Logical 1 [])) ()) (= (Var 4 b) (ConstantReal 5.500000 (Real 8 [])) ())] Source Public Implementation () .false. .false.), test_boz: (Subroutine (SymbolTable 2 {b: (Variable 2 b Local () () Default (Character 1 -2 () []) Source Public Required .false.)}) test_boz [] [(= (Var 2 b) (ConstantString "0b101" (Character 1 1 () [])) ()) (= (Var 2 b) (ConstantString "0b1000000" (Character 1 1 () [])) ()) (= (Var 2 b) (ConstantString "0o10" (Character 1 1 () [])) ()) (= (Var 2 b) (ConstantString "0o70" (Character 1 1 () [])) ()) (= (Var 2 b) (ConstantString "0x2a" (Character 1 1 () [])) ()) (= (Var 2 b) (ConstantString "0xc0ffee" (Character 1 1 () [])) ())] Source Public Implementation () .false. .false.), test_ord_chr: (Subroutine (SymbolTable 3 {a: (Variable 3 a Local () () Default (Integer 4 []) Source Public Required .false.), s: (Variable 3 s Local () () Default (Character 1 -2 () []) Source Public Required .false.)}) test_ord_chr [] [(= (Var 3 a) (ConstantInteger 53 (Integer 4 [])) ()) (= (Var 3 s) (ConstantString "+" (Character 1 1 () [])) ())] Source Public Implementation () .false. .false.)}) [])
